src/js/components/StartupError.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react"
+import styled from "styled-components"
+import {remote} from "electron"
+import Link from "./common/Link"
+import ToolbarButton from "./Toolbar/Button"
+
+const Wrap = styled.div`
+  padding: 24px;
+  padding-top: 54px;
+  min-height: 100vh;
+  background: linear-gradient(to bottom right, #f8eee3, #fae1e0);
```

Comment:
want to use a css var here? This gradient will also be used again in the connection detail designs iirc

Reply:
Yeah good point.